### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by ravenstorm2001**
## 🚀 Description

Pushing some progress I had before I stopped to work on ADS assignment. Further updates to come next week, when I do some more coding.

Please explain the changes you have made.

Major changes:

- Addition of Notebook

### CODE REVIEW

**Comment by Croydon-Brixton** [comment]
@ravenstorm2001 to avoid cluttering github, I suggest you add `__pycache__`, etc. to your `.gitignore` file.

c.f. here for more info: https://git-scm.com/docs/gitignore

File: .gitignore [hunk 1]
```diff
@@ -1,99 +1,4 @@
-# Byte-compiled / optimized / DLL files
-__pycache__/
-*.py[cod]
-
-# C extensions
-*.so
```
Comment (Croydon-Brixton) [inline .gitignore:6]: I suggest you add `__pychache__` to your gitignore 😉

File: src/DataLoaders.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import numpy as np
+import biotite
+import biotite.structure as struc
+import src.DataProcessing  as dp
+from progressbar import ProgressBar
+from typing import Optional
+
```
Comment (Croydon-Brixton) [inline src/DataLoaders.py:62]: As a general note, it's best to subclass the pytorch DataLoaders for the dataloading (equivalently for the datasets). 

You can read more on pytorch datasets & dataloaders here:
https://pytorch.org/tutorials/beginner/basics/data_tutorial.html

As you know, for the geometric setting, the batching works a bit differently to the standard setting, so pytorch geometric provides its own DataLoader class that you can subclass for a graph-based dataloader:
https://pytorch-geometric.readthedocs.io/en/latest/modules/loader.html#torch_geometric.loader.DataLoader

Comment (Croydon-Brixton) [reply src/DataLoaders.py:62]: The dataloaders can then allow you to use multiprocessing (the `num_workers` argument) and useful utils like `pin_memory`

File: src/DataLoaders.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import numpy as np
+import biotite
+import biotite.structure as struc
+import src.DataProcessing  as dp
+from progressbar import ProgressBar
```
Comment (Croydon-Brixton) [inline src/DataLoaders.py]: I recommend using the library [tqdm](https://tqdm.github.io/) for progress bars 😉

File: src/DataLoaders.py (hunk 2 above)
Comment (ravenstorm2001) [reply src/DataLoaders.py:62]: I have commited some change, but am not sure if that is sufficient, could you take a look at that as well? As in the main question is whether my model should have tensors in Dataset itself or a normal list of torch_geometric.data.Data object is fine? And the other thing is whether my preprocessing in init is fine? Would that take a lot of memory if executed like that? The only other alternative is to execute when I start training, but that will then slow down the training by a large factor.

**Pull request closed without merging**